### PR TITLE
Merge staging into master

### PR DESCRIPTION
**Pull request by darkdh**
branch generated by 
1. forking current staging branch
2. push a commit to prevent changing .travis.xml for production
3. rebase against current master

### CODE REVIEW

**Comment by diracdeltas** [comment]
FYI if you are going to do a sync release, there needs to be a version bump: https://github.com/brave/sync/pull/190

**Comment by darkdh** [comment]
@yan do you know why travis is stuck in infinite `npm run flow`?

File: bundles/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@brave/sync-client",
+  "version": "0.1.0",
```
Comment (diracdeltas) [inline bundles/package.json]: i don't remember if there are any clients currently using this NPM package (as opposed to downloading sync bundle from github releases). if so i can publish an update.

**Comment by diracdeltas** [comment]
@darkdh i remember seeing that; we don't really use flow in a useful way so we can just exclude that from travis

**Comment by darkdh** [comment]
There are two tests failed

**Comment by diracdeltas** [comment]
@darkdh probably needs new AWS credentials; see https://github.com/brave/sync/pull/210/files

@evq or someone in #devops can help you get the new credentials

**Pull request merged**